### DISCUSSION TITLE
SVG can't be viewed within GitHub anymore

### DISCUSSION THREAD

**Discussion by Thomas-Boi** [community/community#5546] [Projects and Issues] [answered]
About a few weeks ago, I was able to view SVGs within the repo by clicking on this button:
![image](https://user-images.githubusercontent.com/43018778/132621934-793f3348-dcd7-4ff6-bc65-fd48465ddde8.png)

However, just a few days ago, I wasn't able to do this anymore. The SVG shown will be empty and blank:
![image](https://user-images.githubusercontent.com/43018778/132622002-d39fc1d9-3c4e-440d-92c6-08087a1e19a7.png)

Inspecting the page shows that the `<img>` tag isn't rendered properly:
![image](https://user-images.githubusercontent.com/43018778/132623426-785c5032-ceec-4c57-8c11-eaf5f46d09b5.png)

I'm not sure what's causing the issue. Playing around with the styling helps a little bit but it's not perfect:
![image](https://user-images.githubusercontent.com/43018778/132623535-e5e827d3-be92-44cc-8c7a-552863b573b3.png)

Viewing file raw works for now so it's not too bad. However, I love the old feature and I'd like to see it stays.

**Comment by el-mapache** [comment, marked as answer]
@Thomas-Boi Hello! 👋 Thanks for raising this issue. My apologies for the delay in responding to you.

I'm going to be looking into this issue, and have some questions for you:

* It _looks_ like you are trying to view the rich output within a diff, probably in a PR. Is this accurate?
* Are you still seeing this issue in the repo in question?
* Is this a public or private repo? If it is the former, would you be willing to share it here? Knowing the repo can help us diagnose the issue. At the very least, we can use the svg you are having issues viewing to assist us in reproducing the issue.

As of today, from a pull request, I can view the rich diffs for a number of svgs in our testing repo. Given that, I want to make sure I'm following all of the steps you took to encounter this bug.

Thanks!

**Reply by Thomas-Boi** [reply]
Hi @el-mapache,

So far, it seems that this bug has been fixed. It was in a public repo and we didn't change anything in terms of repo setting. However, thank you for confirming the issue with me.

**Reply by el-mapache** [reply]
Thats great to hear! We were actively making a lot of changes in this area of the codebase at the time you logged this issue, and I'm glad those changes haven't continued to negatively impact your experience.

cheers!
